api/geth_backend.go

```diff
@@ -2658,6 +2658,17 @@ func (b *GethStatusBackend) injectAccountsIntoWakuService(w types.WakuKeyManager
 		b.statusNode.ChatService(accDB).Init(messenger)
 		b.statusNode.EnsService().Init(messenger.SyncEnsNamesWithDispatchMessage)
 		b.statusNode.CommunityTokensService().Init(messenger)
+
+		if messenger != nil && b.statusNode.WakuV2Service() != nil {
+			var deviceType string
+			for _, inst := range messenger.Installations() {
+				if inst.ID == messenger.InstallationID() {
+					deviceType = inst.InstallationMetadata.DeviceType
+					break
+				}
+			}
```

Comment:
Please add `messenger.InstallationDeviceType()` method. Probably reuse `getOurInstallationMetadata` internally.
Looping through array here is not good, knowing that internally it's a `map`.

Reply:
Was able to replace with a call to `m.allInstallations.Load`